dist/clipboard.js

```diff
@@ -26,8 +28,11 @@ if (Element && !Element.prototype.matches) {
  * @return {Function}
  */
 function closest (element, selector) {
-    while (element && element !== document) {
-        if (element.matches(selector)) return element;
+    while (element && element.nodeType !== DOCUMENT_NODE_TYPE) {
+        if (typeof element.matches === 'function' &&
+            element.matches(selector)) {
+          return element;
```

Comment:
This indentation is a little whack to me, though I'm not sure how I'd improve it. I guess I'd prefer a longer line or deeper indented body?